### ISSUE TITLE
Invalid regular expression

### ISSUE THREAD

**Issue by jmontoyaa** [crlcu/multiselect#164]
When searching for a string containing "[" or ")" a js error appears:

multiselect.min.js:11 Uncaught SyntaxError: Invalid regular expression: / [item/: Unterminated character class

A possible solution will be to escape the string like in here:

https://stackoverflow.com/questions/3446170/escape-string-for-use-in-javascript-regex

```
function escapeRegExp(str) {
  return str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "\\$&");
}
```

![screenshot from 2018-07-09 08-24-22](https://user-images.githubusercontent.com/158935/42434641-35b94ef6-8354-11e8-878c-4cad745158d8.png)

**Comment by crlcu** [comment]
Hi @jmontoyaa 

This issue should be fixed now by the version 2.5.2

**Comment by jmontoyaa** [comment]
I confirm the fix works, great! Thanks @crlcu
